nikola/conf.py.in

```diff
@@ -343,6 +356,25 @@ HIDDEN_TAGS = ['mathjax']
 # However, more obscure tags can be hidden from the tag index page.
 # TAGLIST_MINIMUM_POSTS = 1
 
+# A list of dictionaries specifying tags which translate to each other.
+# For example:
+#   [
+#     {'en': 'car', 'de': 'Auto'},
+#     {'en': 'window', 'fr': 'fenêtre'},
+#   ]
+# Note that tags in the same language can appear more than once, for
+# example if they have different meaning:
+#   [
+#     {'en': 'cloud', 'de': 'Wolke'},  # the fluffy thing in the sky
+#     {'en': 'cloud', 'de': 'Cloud'},  # cloud computing
```

Comment:
That was just the first example which came to my mind :) It could also be the other way around.

The problem is that when you're writing a blog over a longer time, you sooner or later have cases where you use one tag for two different things because the word has multiple meanings. If you start translating stuff, you can end up having that tag split up into two different tags.

While this kind of sucks, I'd really do *not* want to disallow this. If a user wants to use the same tag for different things (for example, because he has already been doing that for years and only recently noticed), then why not let him do that?

What we can do is not mention this explicitly in a comment, though, if you'd prefer that.